### ISSUE TITLE
commit for #2652 breaks translations

### ISSUE THREAD

**Issue by ljmerza** [home-assistant/home-assistant-polymer#2660]
**Home Assistant release with the issue:**

latest master

**Last working Home Assistant release (if known):**
reverting commit 4921686bdf109f20d2e5d78f55cf381942f1187b fixes this

**Browser and Operating System:**

Ubuntu 18.10
**Description of problem:**

Getting 404 on /static/translations/en-e0edf001610953a7950474bc91990dfe.json which is most likely causing error below
**Javascript errors shown in the web inspector (if applicable):**

```
Uncaught (in promise) SyntaxError: Unexpected token : in JSON at position 3 translations-mixin.ts:69 
  | async function (async) |   |  
-- | -- | -- | --
  | _loadResources | @ | translations-mixin.ts:68
  | firstUpdated | @ | translations-mixin.ts:21
  | firstUpdated | @ | themes-mixin.js:8
  | firstUpdated | @ | auth-mixin.js:10
  | firstUpdated | @ | home-assistant.ts:83
  | performUpdate | @ | updating-element.ts:660
  | _enqueueUpdate | @ | updating-element.ts:616
  | async function (async) |   |  
  | _enqueueUpdate | @ | updating-element.ts:610
  | requestUpdate | @ | updating-element.ts:594
  | connectedCallback | @ | updating-element.ts:486
  | connectedCallback | @ | lit-element.ts:181
  | ./src/layouts/app/home-assistant.ts | @ | home-assistant.ts:121
  | __webpack_require__ | @ | bootstrap:63
  | ./src/entrypoints/app.js | @ | app.js:1
  | __webpack_require__ | @ | bootstrap:63
  | (anonymous) | @ | bootstrap:195
  | (anonymous) | @ | bootstrap:195



```

**Additional information:**

**Comment by ljmerza** [comment]
Weird, I used the .nvmrc file to get node 8.11.1. I tried removing my node_modules and running script/bootstrap again but the problem still persists. My cache is disabled and the service worker is set to bypass for network as well. I tried clearing out my cache/cookies/etc but all of the localization is still broken. 

This is what I get now at the home screen.

![error message](https://i.imgur.com/6Ixqc57.png)
